Public/Invoke-AtomicTest.ps1

```diff
@@ -193,24 +211,6 @@
                 return
             }
 
-            # Since there might a comma(T1559-1,2,3) Powershell takes it as array.
-            # So converting it back to string.
-            if ($AtomicTechnique -is [array]) {
-                $AtomicTechnique = $AtomicTechnique -join ","
-            }
-
-            # Splitting Atomic Technique short form into technique and test numbers.
-            $AtomicTechniqueParams = ($AtomicTechnique -split '-')
-            $AtomicTechnique = $AtomicTechniqueParams[0]
-
-            if ($AtomicTechniqueParams.Length -gt 1) {
-                $ShortTestNumbers = $AtomicTechniqueParams[-1]
-            }
-
-            if ($null -eq $TestNumbers -and $null -ne $ShortTestNumbers) {
-                $TestNumbers = $ShortTestNumbers -split ','
-            }
-
```

Comment:
The cod for converting short form to TestNumbers lies within the `isLoggingModuleSet` condition. This should happen regardless of whether LoggingModule has been defined.